Cargo.toml

```diff
@@ -386,6 +386,7 @@ ntapi = { git = "https://github.com/MSxDOS/ntapi.git", rev = "24fc1e47677fc9f6e3
 # version of that crate with the appropriate dependency patched in.
 openssl-sys = { git = "https://github.com/vectordotdev/rust-openssl.git", tag = "openssl-sys-v0.9.91+3.0.0" }
 openssl-src = { git = "https://github.com/vectordotdev/openssl-src-rs.git", tag = "release-300-force-engine+3.1.2"}
+ed25519-dalek = { git= "https://github.com/dalek-cryptography/ed25519-dalek.git", tag = "2.0.0-rc.3", default-features = false, features = ["u64_backend"] }
```

Comment:
This should include a comment saying why it needed to be patched / when it can be removed (preferably containing a link to an issue that can be tracked)

Reply:
Also: are there any known issues in 2.0.0-rc.3 that are fixed in 2.0.0? Just wondering if those issues are more or less severe than this particular issue (in which case we may want to add an ignore entry instead).

Reply:
As discussed [here](https://dd.slack.com/archives/C03HCL98TM2/p1692043048411659), re-purposed this PR to just ignore this vulnerability.